cli/ops/fs.rs

```diff
@@ -15,10 +15,10 @@ use std::fs;
 use std::io::SeekFrom;
 use std::path::Path;
 use std::time::UNIX_EPOCH;
-use tokio;
+use tokio::fs as tokio_fs;
```

Comment:
tokio::fs is one character shorter than the tokio_fs. This alias just adds complexity (admittedly a very small amount, but still more complex than not adding alias.)

Reply:
Sure, will delete. It made more sense in the context of the whole batch of commits I'm now pushing piecemeal, since there are some functions using std::fs, some using tokio::fs, and some using cli/fs.rs. I thought it'd be easier to keep track by referring to them as fs, tokio_fs, and deno_fs, respectively. But it's not clear that things will look the same in the end anyway, as these PRs get refined and merged.

Will push commits making all these revisions shortly.